components/desktop/gnome2/gnome-games/Makefile

```diff
@@ -29,7 +30,8 @@ include $(WS_TOP)/make-rules/prep.mk
 include $(WS_TOP)/make-rules/configure.mk
 include $(WS_TOP)/make-rules/ips.mk
 
-PATH=/usr/gnu/bin:/usr/bin
+# Need automake 1.11
```

Comment:
I presume you are objecting to the comment.  Indeed, it is misleading and incorrect in most cases.  I'll remove all of those comments and produce a new PR.